packages/mwp-config/README.md

```diff
@@ -21,16 +21,10 @@ is used by many different packages. For the most part, they describe the
 application environment, e.g. standard file `paths`, `env` values, and `package`
 config values.
 
-## Build config
-
-Build-time config is primarily used by Webpack and its dependencies to define
-build behavior, including paths to the source files and build targets.
+## Babel config
```

Comment:
we should probably move the babel config over to mwp-cli as well since it is only used there. followup.